attend-events.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```

Comment:
I'm not quite sure how to comment on the file as a whole, but just wanted to point out that you have this file in the root folder. Unfortunately Gatsby won't be able to see this file, so it won't actually create a page for it. Can you move this file to the `src/pages` directory? Gatsby will then be able to pick it up, work some magic ✨ and create a page for it. 

From what I can tell otherwise, this page looks pretty good 🙂